src/sdk/pynni/nni/gridsearch_tuner/gridsearch_tuner.py

```diff
@@ -95,19 +94,16 @@ def json2parameter(self, ss_spec):
 
     def _parse_quniform(self, param_value):
         '''parse type of quniform parameter and return a list'''
-        if param_value[2] < 2:
-            raise RuntimeError("The number of values sampled (q) should be at least 2")
-        low, high, count = param_value[0], param_value[1], param_value[2]
-        interval = (high - low) / (count - 1)
-        return [float(low + interval * i) for i in range(count)]
+        low, high, interval = param_value[0], param_value[1], param_value[2]
+        count = int(np.floor((high - low) / interval)) + 1
+        return [low + interval * i for i in range(count)]
 
     def parse_qtype(self, param_type, param_value):
         '''parse type of quniform or qloguniform'''
-        if param_type == 'quniform':
-            return self._parse_quniform(param_value)
-        if param_type == 'qloguniform':
-            param_value[:2] = np.log(param_value[:2])
-            return list(np.exp(self._parse_quniform(param_value)))
+        if param_type in ['quniform', 'qloguniform']: 
```

Comment:
cannot deal with quniform and qloguniform in the same way

Reply:
what's the difference ?